src/constants/index.js

```diff
@@ -2,9 +2,31 @@ export const DEFAULT_REGISTRATION_NAME = "John Doe";
 export const DEFAULT_REGISTRATION_EMAIL = "john.doe@example.com";
 export const DEFAULT_REGISTRATION_PASSWORD = "adMIN1234!";
 
-export const DEFAULT_LOGIN_EMAIL = "lucas.douglas@example.org";
+export const DEFAULT_LOGIN_EMAIL = "leanne.lindgren@example.org";
 export const DEFAULT_LOGIN_PASSWORD = "adMIN1234!";
 
+export const MOVIE_GENRES = {
+    comedy: "comedy",
+    fantasy: "fantasy",
+    scifi: "sci fi",
+    horror: "horror",
+    thriller: "thriller",
+};
+
+export const DEFAULT_MOVIE_TITLE = "Matrix";
+export const DEFAULT_MOVIE_DESCRIPTION =
+    "Thomas Anderson, a computer programmer, is led to fight an underground war against powerful computers who have constructed his entire reality with a system called the Matrix.";
+export const DEFAULT_MOVIE_IMAGEURL =
+    "https://flxt.tmsimg.com/assets/p22804_p_v8_av.jpg";
+export const DEFAULT_MOVIE_GENRE = MOVIE_GENRES.scifi;
+
+export const DEFAULT_COMMENT = "Awesome movie!";
+
+export const URL_REGEX =
+    /^((https?|ftp):\/\/)?(www.)?(((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:)*@)?(((\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.(\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.(\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5])\.(\d|[1-9]\d|1\d\d|2[0-4]\d|25[0-5]))|((([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.)+(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.?)(:\d*)?)(\/((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)+(\/(([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)*)*)?)?(\?((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)|[\uE000-\uF8FF]|\/|\?)*)?(\#((([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(%[\da-f]{2})|[!\$&'\(\)\*\+,;=]|:|@)|\/|\?)*)?$/i;
```

Comment:
Mati moja 😨

Reply:
Ovo koristis za validaciju pretpostavljam. Validaciju bi trebalo raditi na BE jer to je zapravo mesto gde se validacije mnogo cesce rade, pogotovo kada su ovako kompleksnije kao ova.

Reply:
Da, da :) Mene su ucili do sada da ide na obe strane, i na FE i na BE... Pogledacu da dodam to u Laravelu, a onda ovo da brisem?